### ISSUE TITLE
start command does not start, but console works fine (on debian jessie)

### ISSUE THREAD

**Issue by DylanGriffith** [bitwalker/exrm#233]
I've got an exrm release that is built for and running on a debian jessie machine. When I try to start it using the `start` command it just outputs:

```
forwarder@forwarder:~/app_test$ bin/staging_forwarder start
Using /home/forwarder/app_test/releases/0.0.1/staging_forwarder.sh
```

And when I look at the processes there is nothing created or nothing running. Also it's not doing what it's supposed to do. However when I start it using `console` it works fine. It starts the application and does what it's supposed to do.

Is there any way I can debug what is actually happening when I run `start`? It seems to return a 0 exit code and I can't find any errors or logs anywhere.

The `mix.exs` is:

```elixir
defmodule StagingForwarder.Mixfile do
  use Mix.Project

  def project do
    [app: :staging_forwarder,
     version: "0.0.1",
     elixir: "~> 1.1",
     build_embedded: Mix.env == :prod,
     start_permanent: Mix.env == :prod,
     deps: deps]
  end

  def application do
    [applications: [:logger, :amqp],
     mod: {StagingForwarder, []}]
  end

  defp deps do
    [
      {:exrm, "~> 0.19"},
      {:amqp, "~> 0.1"},
    ]
  end
end
```

**Comment by DylanGriffith** [comment]
Ok so this was actually problem with my machine. The problem was that the user didn't have permission to run the erlang daemon. Specifically the problem was:

```
$ ls -lh /dev/ptmx
c--------- 1 root root 5, 2 Nov  6 03:45 /dev/ptmx
```

And the problem was fixed by `chmod a+rw /dev/ptmx`

Hopefully this serves to help somebody googling for this same problem in the future. I had to do many a google before I figured this out and I realised the problem had nothing to do with exrm when I figured out that I also couldn't run tmux or screen.
